be/src/olap/rowset/alpha_rowset_reader.cpp

```diff
@@ -207,6 +215,16 @@ OLAPStatus AlphaRowsetReader::_get_next_row_for_cumulative_rowset(RowCursor** ro
     OLAPStatus status = _get_next_not_filtered_row(pos, row);
     RowBlock* row_block = _row_blocks[pos];
     row_block->pos_inc();
```

Comment:
_row_blocks[pos]->pos_inc() may be better

Reply:
ok